### ISSUE TITLE
Add support for \xrightarrow

### ISSUE THREAD

**Issue by devon** [Khan/KaTeX#367]
`\xrightarrow` or `\overset` support text above arrow.

![xrightarrow](http://i.stack.imgur.com/MtENj.png)

**Comment by Newman101** [comment]
Just a quick one: Should this be relation or accent? There is already rightarrow defined: 

```javascript 
defineSymbol(math, main, rel, "\u2192", "\\rightarrow");
```

Also, which unicode shall I use? It seems that \\xrightarrow is AMSMath symbol.

**Comment by devon** [comment]
We cannot add letter on the top of rightarrow? Am I right?

Yes, \xrightarrow is AMSMath symbol. You have to add amsmath plugin when use mathjax. 

It should be great if KaTex support this symbol.

**Comment by Newman101** [comment]
So basically if I got this right: adding support of xrightarrow to KaTeX requires u2192 as an accent. If someone can verify this, so I can create a PR.

Thank you.

**Comment by kevinbarabash** [comment]
`\xrightarrow` and `\overset` should be functions because they accepts a parameter.

**Comment by Newman101** [comment]
Ok - what about the unicode? Is it u2192?

**Comment by kevinbarabash** [comment]
@Newman101 yes, `\u2192` is the right character.  The arrow's tail grows to match the content positioned above it.

**Comment by sheljohn** [comment]
@Newman101 Any news about this? Seems like it was on the right track but no news for almost two years :/ Is this being dropped?

**Comment by kevinbarabash** [comment]
There's support for `\xrightarrow` on `master right now.  I need to publish a new version to npm which I haven't got round to yet.
